src/main/java/uk/gov/companieshouse/api/accounts/controller/CurrentPeriodController.java

```diff
@@ -26,10 +39,39 @@
     public ResponseEntity create(@Valid @RequestBody CurrentPeriod currentPeriod,
             HttpServletRequest request)
             throws NoSuchAlgorithmException {
-        Transaction transaction = (Transaction) request.getSession().getAttribute(AttributeName.TRANSACTION.getValue());
+        Transaction transaction = (Transaction) request.getSession()
+                .getAttribute(AttributeName.TRANSACTION.getValue());
         CurrentPeriod result = currentPeriodService
                 .save(currentPeriod, transaction.getCompanyNumber());
         return ResponseEntity.status(HttpStatus.CREATED).body(result);
 
     }
+
+    @GetMapping(value = "/transactions/{transactionId}/company-accounts/{companyAccountId}/small-full/current-period",
```

Comment:
In the web application the request URLs are being moved to class level (see https://github.com/companieshouse/company-accounts.web.ch.gov.uk/pull/22). Can we be consistent with the approach please?